lf/demo/demo_robot.lf

```diff
@@ -2,24 +2,53 @@ target C {
     timeout: 15 sec
 };
 
-reactor Sensors {    
+// This is what we call a driver container. But I think at Luos we will have 2 drivers, one for the button and one for bumper.
+// But perhaps its more lik nodes.
+reactor Sensors {
+    // Those are like pipe beetween reactors?
     output button_out:bool;
     output bumper_out:bool;
     
+    // State are like shared variables?
```

Comment:
You can think of them as private variables internal to a reactor.

Reply:
In LF, although the user can declare global (shared) variables using `preamble`, using them is discouraged as they can generate race conditions that lead to nondeterminism.

Reply:
Ok so you have to link them to "output" to be able to share it?